en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -1,14 +1,19 @@
 # Salesforce Rest API Connector Example
-
+salesforce
 The Salesforce REST Connector allows you to work with records in Salesforce, a web-based service that allows organizations to manage contact relationship management (CRM) data. You can use the Salesforce connector to create, query, retrieve, update, and delete records in your organization's Salesforce data. The connector uses the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm) to interact with Salesforce.
 
 ## What you'll build
 
 This example explains how to use the Salesforce client to connect with the Salesforce instance and perform the 
 following operations:
 
-* Create an Account 
-* Execute a SOQL query to retrieve the Account Name and ID in all the existing Accounts. 
+* Create an Account.
+
+  The user sends the request payload which includes with `sObjects` (Any object that can be stored in the Lightning platform database), to create a new Account object in the sales force. This request is sent to WSO2 EI by invoking the Salesforce connector API. 
```

Comment:
The user sends the request payload that includes `sObjects` (any object that can be stored in the Lightning platform database), to create a new Account object in Salesforce. This request is sent to WSO2 EI by invoking the Salesforce connector API.

Reply:
Done.